lib/ical_filter_proxy/calendar_builder.rb

```diff
@@ -33,5 +34,37 @@ def add_rules
       end
     end
 
+    def add_alarms
+      alarms = calendar_config["alarms"]
+      return unless alarms
+
+      if alarms["clear_existing"] == true
+        calendar.set_clear_existing_alarms
+      end
+
+      triggers = alarms["triggers"]
+      return unless triggers
+
+      triggers.each do |trigger|
+        calendar.add_trigger(generate_iso_format(trigger))
+        end
+    end
+
+    def generate_iso_format(input)
+      if /(\d+)\s+days?/i =~ input
```

Comment:
You could replace all if these if statements with a `case / else` statement.

Reply:
Done.